### DISCUSSION TITLE
Dev Console Error

### DISCUSSION THREAD

**Discussion by babyaceinspace** [gavvvr/obsidian-imgur-plugin#86] [Getting help]
Whenever I try to upload an image, this error comes up on the dev console. I'm getting the feeling it's not something I can fix. 

![dev console error](https://github.com/user-attachments/assets/285543ad-1c19-443b-bcad-89bd79a09e0e)

**Comment by gavvvr** [comment]
Hi @babyaceinspace 
I'm not sure that the message from your screenshot is related to the Imgur plugin. It seems to be just an informational message, not an error.

What happens with the image when you paste it? Is there a message to check dev console? If so, the console should show some error in dangerous red color.

**Reply by babyaceinspace** [reply]
I just skipped taking screen shots. There was an error message telling me to check the dev console. When I checked the dev console for the error related to the obsidian plugin, the screenshot I shared was what came up. 
Moot point though, it's since been fixed somehow???? I can upload images no problem  ¯\_ (ツ)_/¯
